### PR TITLE
Adding LICENSE file

### PR DESCRIPTION
**Pull request by matzew**
@PhilipGough mind reviewing ?

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (philipgough) [inline LICENSE:189]: Want to add anything here?

Comment (matzew) [reply LICENSE:189]: No, it's the standard license file, and it's a note _after_ the actual terms and conditions of the license.  It's basically a "template" / "example" how to license your own work under that license.

Here is a concrete usage of that "template"
https://github.com/aerogear/kafka-cdi/blob/master/src/main/java/org/aerogear/kafka/cdi/annotation/Consumer.java#L2

**Comment by matzew** [comment]
@PhilipGough any more comments ?

**Comment by philipgough** [comment]
@matzew No LGTM lets merge

**Pull request merged**